web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -156,35 +151,63 @@ export const LicenseStatusReceipt = (props: Props): ReactElement => {
+            <UnStyledButton
+              className="tablet:margin-left-1"
+              style="default"
+              isUnderline={true}
+              dataTestid="edit-button"
+              onClick={(): void => {
+                analytics.event.task_status_checklist_edit_button.click.edit_address_form();
+                props.onEdit();
+              }}
+            >
+              {Config.licenseSearchTask.editButtonText}
+            </UnStyledButton>
+          </div>
+        </div>
+
+        <div className="border-2px border-white radius-lg bg-white padding-y-2 padding-x-105 margin-bottom-5 margin-top-205 shadow-3">
           <div
-            className={`${theme.bgColor} ${theme.borderColor} padding-3 border-top-2px font-body-md text-bold fdr fac`}
+            className={`margin-1 text-bold fdc fac radius-lg ${permitColorScheme.bgSubHdrColor}`}
             data-testid={`permit-${props.status}`}
           >
-            <span className="padding-right-1">{Config.licenseSearchTask.permitStatusText}</span>
-            <Icon className={`${theme.headerIconColor} usa-icon--size-3`}>{getIcon(props.status)}</Icon>
-            <span className={`${theme.textColor} padding-left-05 text-uppercase`}>{getText()}</span>
-          </div>
-          <div className="margin-3 font-body-2xs">
-            <div className="fdr">
-              <div className="text-bold">{business?.licenseData?.nameAndAddress.name}</div>
-              <button
-                data-testid="edit-button"
-                onClick={(): void => {
-                  analytics.event.task_status_checklist_edit_button.click.edit_address_form();
-                  props.onEdit();
-                }}
-                className="usa-button usa-button--unstyled mla font-body-2xs underline width-auto"
-              >
-                {Config.licenseSearchTask.editButtonText}
-              </button>
+            <div
+              className={`${permitColorScheme.bgHdrColor} text-white width-full radius-top-lg padding-y-1 padding-x-2 margin-0`}
+            >
+              {Config.licenseSearchTask.permitStatusText}
             </div>
-            <div className="border-dashed border-bottom-2px border-top-0 border-left-0 border-right-0 border-base-lighter padding-bottom-3">
-              {getOneLineAddress()}
+
+            <div
+              className={`padding-05 fdr fac radius-bottom-lg ${permitColorScheme.textAndIconColor} width-full font-sans-lg text-bold`}
+            >
+              <Icon className="display-none tablet:display-block tablet:margin-left-1 usa-icon--size-4">
+                {getIcon(props.status)}
+              </Icon>
+
+              <p className="tablet:margin-left-1 padding-05 line-height-sans-3">{getText()}</p>
             </div>
           </div>
-          {props.items.map(receiptItem)}
+          <div className="padding-2">
+            <Heading level={1} styleVariant="h4" className="margin-bottom-05-override tablet:padding-top-1">
```

Comment:
[dust] Love to see me a `<Heading />`! Just checking though, the desired behavior is a heading level of 1 and a style of `h4-styling`, correct?

Reply:
It's an `<h4>` in the designs, so I added it here as such, but got feedback about it being an accessibility issue since an `<h1>`, `<h2>`, and `<h3>` do not render ahead of the `<h4>`. So I updated it to `<Heading level={1} styleVariant="h4"...` so as not to cause any accessibility issues since there was initially an absence of the higher level tags.